pkgs/shells/mrsh/default.nix

```
@@ -1,14 +1,14 @@
 { lib, stdenv, fetchFromGitHub, meson, ninja, pkg-config, readline }:
 
 stdenv.mkDerivation rec {
-  pname   = "mrsh";
-  version = "2020-11-04";
+  pname = "mrsh-unstable";
+  version = "2021-01-10";
```

```suggestion
  pname = "mrsh";
  version = "unstable-2021-01-10";
```

There is only one version of mrsh in nixpkgs.